### ISSUE TITLE
Cannot scroll to bottom with chrome

### ISSUE THREAD

**Issue by keithc-ca** [eclipse-openj9/openj9-docs#877]
When the browser window is wide enough, the menu is expanded inline on the left side of the page. After expanding to a long submenu (e.g. 'JVM -X options') no scrollbar appears and using the mouse wheel the menu stops scrolling before reaching the bottom. Selecting the last visible entry makes a few more entries accessible, but you have to apply that workaround multiple times to reach some of the final entries.

Along the way, the main footer scrolls up beyond the top of the screen as you try to reach the bottom of the menu.

Observed in chrome version 97.0.4692.71 (Official Build) (64-bit). 

Scrolling behavior works as expected with these 64-bit browsers
* brave 97.0.4692.71
* edge 97.0.1072.64
* firefox version 96.0.1
This is surprising given that both brave and edge are built upon the same engine as chrome.

**Comment by Sreekala-Gopakumar** [comment]
@keithc-ca - Is this still persisting? Are you able to reproduce this issue? I couldn't see this issue. Wanted to check whether it got resolved because of mkdocs builds update. Thanks!

**Comment by keithc-ca** [comment]
Yes, I encountered it again the other day. It seems to be dependent on the width of the browser window; if I maximize the window, then the scrollbar appears.

**Comment by doveye** [comment]
I just had a play and I can make it behave oddly too 🙂. If I reduce the width of the browser window until the left-nav disappears (which is working as designed - you get a hamburger menu at the top instead), then slowly and gradually increase the browser window width, I can get a sort of halfway point where the left nav is there (but fully expanded, which is odd) and so is the hamburger menu (but it doesn't do anything when you click the icon), but the scrollbar is absent. I wasn't using a mouse so couldn't test the wheel scroll issue but there is definitely something strange in the left nav behaviour. 

![image](https://user-images.githubusercontent.com/14108229/232773685-c73cc1cf-4304-4d39-bd31-f310bbbfa08b.png)

Should look either like this (narrow window, no left nav, hamburger menu):

![image](https://user-images.githubusercontent.com/14108229/232772800-0bf15607-6a56-4667-8923-94e57207de48.png)

Or like this (wide window, left nav with collapsed entries and scroll bar, no hamburger menu):

![image](https://user-images.githubusercontent.com/14108229/232773479-6f48f573-16be-423a-89ef-09e9848408f7.png)

I'm using Chrome 112.0.5615.121 on Windows 10.

**Comment by doveye** [comment]
I can reproduce the above behaviour on Firefox and Edge. Next step would be to try a vanilla MkDocs+Material and see if that shows the same behaviour.
